### ISSUE TITLE
auto-flush needs to happen at a higher time interval

### ISSUE THREAD

**Issue by grosser** [DataDog/dogstatsd-ruby#238]
we switched v4 -> v5 and got lots of no-data alerts
the sender thread should make sure to send a batch every 1min for example to avoid this pitfall

**Comment by januszm** [comment]
it sounds like it's too early to switch to v5? @grosser  did you rollback to v4 after discovering this?

**Comment by grosser** [comment]
no, I run v5 with single_thread + times flush + manual flushing after large batches

**Comment by grosser** [comment]
don't remember, sounds like a useful setting though
